### PR TITLE
fix(issue): fix 166

### PR DESCRIPTION
**Pull request by pengYYYYY**
#166

### CODE REVIEW

File: src/cascader/utils/inputContent.ts [hunk 1]
```diff
@@ -150,7 +150,11 @@ export function outerClickListenerEffect(
   event: MouseEvent | TouchEvent,
 ) {
   const { visible, setVisible } = cascaderContext;
-  if (!ref || ref.contains(event.target as Node)) {
+  if (
+    !ref ||
+    ref.contains(event.target as Node) ||
+    Array.from((event.target as HTMLElement)?.classList).includes('t-tag')
```
Comment (LeeJim) [inline src/cascader/utils/inputContent.ts:156]: 可以直接用 `classList.contains`

**Review by LeeJim — CHANGES_REQUESTED** [review]

File: src/cascader/utils/inputContent.ts (hunk 1 above)
Comment (pengYYYYY) [reply src/cascader/utils/inputContent.ts:156]: image

Comment (pengYYYYY) [reply src/cascader/utils/inputContent.ts]: 已改https://github.com/Tencent/tdesign-vue-next/pull/231/commits/9f8c796c6ed3e5e3bc08752a4c8fefb94017a549

**Review by LeeJim — APPROVED** [review]
LGTM

**Pull request merged**